java/registry/src/main/java/dev/sunbirdrc/registry/helper/RegistryHelper.java

```diff
@@ -191,6 +193,49 @@ public JsonNode removeFormatAttr(JsonNode requestBody) {
         return requestBody;
     }
 
+    /**
+    * REUSBALE METHOD FOR POST API CALLS
+     */
+    public void apiHelper(JsonNode obj,String url){
+         WebClient.Builder builder = WebClient.builder();
+        try{
+            Mono<JsonNode> responseMono = builder.build()
+                    .post()
+                    .uri(url)
+                    .contentType(MediaType.APPLICATION_JSON)
+                    .accept(MediaType.APPLICATION_JSON)
+                    .bodyValue(obj)
+                    .retrieve()
+                    .bodyToMono(JsonNode.class)
+                    .onErrorResume(throwable -> {
+                        throwable.printStackTrace();
+                        return Mono.empty();
+                    });
+
+            JsonNode response = responseMono.block();
+            logger.info("RESPONSE  {}",response);
+        }catch(Exception e){
+            logger.error("Exception occurred !" , e);
+        }
+    }
+
+    /**
+     * Anchors schema to the CORD CHAIN
+     */
+    public void anchorSchemaAPI(JsonNode obj){
+        // apiHelper(obj,"http://172.24.0.1:5106/api/v1/schema");
+        apiHelper(obj,"http://localhost:5106/api/v1/schema"); // considering issuer agent running in local
```

Comment:
this should be a ENV variable, which can be changed. Say `ISSUER_AGENT_URL` or something like that.